<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widerstandsteiler</class>
 <widget class="QMainWindow" name="Widerstandsteiler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widerstandsteiler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QLabel" name="Tolleranz">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tolleranz</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>51</y>
      <width>202</width>
      <height>224</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="UIn">
           <property name="text">
            <string>U In</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="UinStr"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="UOut">
           <property name="text">
            <string>U Out</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="UoutStr"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="axWiderstand">
         <property name="text">
          <string>Max Widerstand</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="MaxResStr"/>
         </item>
         <item>
          <widget class="QSpinBox" name="MaxResExp"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="EReihe">
         <property name="text">
          <string>Norm Reihe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="NormReihe">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="prefix">
          <string>E</string>
         </property>
         <property name="EReihen" stdset="0">
          <stringlist>
           <string>E3</string>
           <string>E6</string>
           <string>E12</string>
           <string>E24</string>
           <string>E48</string>
           <string>E96</string>
           <string>E192</string>
          </stringlist>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>181</width>
      <height>270</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="Picture">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Images/Widerstand.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="Res1">
         <property name="text">
          <string>Res1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Res2">
         <property name="text">
          <string>Res2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
    <zorder>Picture</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>UinStr</tabstop>
  <tabstop>UoutStr</tabstop>
  <tabstop>MaxResStr</tabstop>
  <tabstop>MaxResExp</tabstop>
  <tabstop>NormReihe</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
